app/content/publication-api.md

```diff
@@ -65,6 +65,23 @@ If you don't have a subscription to the publication that this demo application i
         <p>Use the Publication API's <code>entitlementsPlans</code> endpoint to query the full detail of an entitlement for a logged-in user.</p>
       </td>
     </tr>
+    <tr id="member" class="hidden">
```

Comment:
Given this uses a completely different authentication, do you think it makes sense to move this to a different page?  Or at least distinguish it further on the page by calling this "Monetization API" to align with the previous discussion?

Reply:
@ed-bird Yeah, I agree. The reason I was avoiding it here is that while we do have the RRM:E client generated, I need to re-generate it to import the new endpoints, so these examples are still using the `access_token`. I'll do the work to regenerate it and separate these out, that's a good call.